Buildings/Controls/OBC/FDE/PackagedRTUs/BAScontroller.mo

```diff
@@ -0,0 +1,220 @@
+
+  parameter Real minDDSPset(
+  final unit="Pa",
+  final displayUnit="Pa",
+  final quantity="PressureDifference")=1.25e-3
+  "Minimum down duct static pressure reset value"
+  annotation (Dialog(tab="Supply fan", group="Down duct pressure limits"));
+
+  // ---inputs---
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanInput occ
+    "True when occupied mode is active"
+    annotation (Placement(transformation(extent={{-140,60},{-100,100}})));
+  Buildings.Controls.OBC.CDL.Interfaces.IntegerInput occReq
+    "Terminal unit occupancy requests"
+    annotation (Placement(transformation(extent={{-140,30},{-100,70}})));
+  Buildings.Controls.OBC.CDL.Interfaces.IntegerInput sbcReq
+    "Terminal unit setback cooling requests"
+    annotation (Placement(transformation(extent={{-140,-28},{-100,12}})));
+  Buildings.Controls.OBC.CDL.Interfaces.IntegerInput sbhReq
+    "Terminal unit setback heating requests"
+    annotation (Placement(
+        transformation(extent={{-140,-58},{-100,-18}})));
+  Buildings.Controls.OBC.CDL.Interfaces.IntegerInput totCoolReqs
+    "Total terminal unit cooling requests"
+    annotation (Placement(
+        transformation(extent={{-140,-88},{-100,-48}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput highSpaceT
+    "Highest space temperature reported from all terminal units"
+    annotation (Placement(transformation(extent={{-140,-124},{-100,-84}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput mostOpenDam
+    "Most open damper position of all terminal units" annotation (
+      Placement(transformation(extent={{-140,-156},{-100,-116}})));
+
+  // ---outputs---
+  output Buildings.Controls.OBC.CDL.Interfaces.BooleanOutput yOcc
+    "True when occupied mode is active"
+    annotation (Placement(transformation(extent={{104,28},{144,68}})));
+  output Buildings.Controls.OBC.CDL.Interfaces.BooleanOutput ySBC
+    "True when setback cooling mode is active"
+    annotation (Placement(transformation(extent={{104,-2},{144,38}})));
+  output Buildings.Controls.OBC.CDL.Interfaces.BooleanOutput ySBH
+    "True when setback heating mode is active"
+    annotation (Placement(transformation(extent={{104,-36},{144,4}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealOutput yMinOAflowSet
+    "active outside air flow set point sent to factory controller"
+    annotation (Placement(transformation(extent={{104,56},{144,96}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealOutput yBldgSPset
+    "Building static pressure set point"
+    annotation (Placement(transformation(extent={{104,-112},{144,-72}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealOutput yDDSPset
+    "Calculated down duct static pressure set point"
+    annotation (
+      Placement(transformation(extent={{104,-154},{144,-114}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealOutput ySATset
+    "Calculated supply air temperature set point"
+    annotation (Placement(transformation(extent={{104,-74},{144,-34}})));
+
+  TSupSet tSupSet
```

Comment:
Please use full path `Buildings.Controls.OBC.FDE.PackagedRTUs.TSupSet`. 

Also, changing the class name from `TSupSet` to `SupplyTemperature`. So here it will be:
```
Buildings.Controls.OBC.FDE.PackagedRTUs.SupplyTemperature supTemSet
```